Makefile

```diff
@@ -28,9 +28,9 @@ SMF_MANIFESTS_IN =	smf/manifests/single-binder.xml.in \
 #
 # Variables
 #
-NODE_PREBUILT_TAG =		zone
-NODE_PREBUILT_VERSION :=	v0.12.9
-NODE_PREBUILT_IMAGE =		fd2cc906-8938-11e3-beab-4359c665ac99
+NODE_PREBUILT_TAG =		zone64
+NODE_PREBUILT_VERSION :=	v8.17.0
+NODE_PREBUILT_IMAGE =		5417ab20-3156-11ea-8b19-2b66f5e7a439
```

Comment:
You could consider taking this Makefile patch that will drop bothering to compile a local node v8 from source when running `make all` on non-SmartOS:

```
diff --git a/Makefile b/Makefile
index e866fbc..d0cbe0d 100644
--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,10 @@ ifeq ($(shell uname -s),SunOS)
         include ./deps/eng/tools/mk/Makefile.node_prebuilt.defs
         include ./deps/eng/tools/mk/Makefile.agent_prebuilt.defs
 else
-        include ./deps/eng/tools/mk/Makefile.node.defs
+       NPM=npm
+       NODE=node
+       NPM_EXEC=$(shell which npm)
+       NODE_EXEC=$(shell which node)
 endif
 include ./deps/eng/tools/mk/Makefile.node_modules.defs
 include ./deps/eng/tools/mk/Makefile.smf.defs
```

This is what other repos tend to do. There is no real value in our eng/tools/mk/Makefile.node.* makefiles.

Reply:
Done.